Octo-Bot/healthcare-registration-bot/build.sh

```diff
@@ -0,0 +1,3 @@
+chmod a+x *.sh &&
+
+sudo docker build . -t healthcare-registration-bot:latest
```

Comment:
Add a newline at the last line.
```suggestion
sudo docker build . -t healthcare-registration-bot:latest

```

Reply:
Resolved!